charts/sn-platform/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -25,12 +25,18 @@ metadata:
   namespace: {{ template "pulsar.namespace" . }}
   annotations:
     "cloud.streamnative.io/enable-config-prefix": "false"
+    {{- with .Values.bookkeeper.bookKeeperCluster.annotations -}}
+    {{ toYaml . | nindent 4 }}
+    {{- end }}
   labels:
     {{- include "pulsar.standardLabels" . | nindent 4 }}
     component: {{ .Values.bookkeeper.component }}
 {{- with .Values.bookkeeper.labels }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+    {{- with .Values.bookkeeper.bookKeeperCluster.labels -}}
```

Comment:
How about using the same indent format of `.Values.bookkeeper.labels` to make the template neat?

Reply:
Fixed